src/_common/content/content-viewer/components/AppContentViewerParagraph.vue

```diff
@@ -0,0 +1,28 @@
+<script lang="ts" setup>
+import { computed, PropType, toRefs } from 'vue';
+import { ContentObject } from '../../content-object';
+import { useContentOwnerController } from '../../content-owner';
+import { renderChildren } from './AppContentViewerBaseComponent.vue';
+
+const props = defineProps({
+	contentData: {
+		type: Object as PropType<ContentObject>,
+		required: true,
+	},
+});
+
+const { contentData } = toRefs(props);
+
+const owner = useContentOwnerController()!;
+
+const rootComponent = computed(() => (owner.contentRules.inlineParagraphs ? 'span' : 'p'));
```

Comment:
oddly enough, I think there's some restrictions on what can go inside certain tags. span is HTML-defined inline, so it doesn't like putting block level within it. i think to be safe we should actually just use a div, or even still a p tag, and override the styling with "display: inline"
https://stackoverflow.com/questions/11314668/what-elements-can-a-span-tag-contain-in-html5

Reply:
Changed to use the `p` tag with `display: inline`